docker-compose/traefik/openwifi_selfsigned.yaml

```diff
@@ -64,6 +71,14 @@ tcp:
       loadBalancer:
         servers:
           - address: "owprov.wlan.local:16005"
+    owprov-restapi:
```

Comment:
This entry is duplicate.

Reply:
Yep, copied too much on owsub addition. Thanks for noticing!